misc/generate_taichi_unreal_header.py

```diff
@@ -0,0 +1,53 @@
+# Generated by GPT-4 with minor modifications
+# Wraps C-API funciton calls with CALL_C_API_FUNC macro
+
+import os
+import re
+
+
+def process_file(file_path, new_file_path):
+    with open(file_path, 'r') as f:
+        lines = f.readlines()
+
+    include_section_end = 0
+    for index, line in enumerate(lines):
+        if line.startswith('#include'):
+            include_section_end = index
+
+    macro_def = '''
+#include "AOTPlugin.h"
```

Comment:
Is this general enough?

Reply:
Nope, it's actually designed to be used in combination with: https://github.com/jim19930609/taichi-unreal. Or maybe I should put this script in the taichi-unreal repo?